aspnetcore/blazor/index.md

```diff
@@ -165,16 +84,22 @@ After rendering is complete, interactivity on the client can adopt either or bot
   * The assemblies and the [.NET runtime](/dotnet/framework/get-started/overview) are downloaded to the browser.
   * Blazor bootstraps the .NET runtime and configures the runtime to load the assemblies for the app. The Blazor WebAssembly runtime uses JavaScript interop to handle DOM manipulation and browser API calls.
 
-<!--
+Blazor apps can entirely target running on WebAssembly in the browser without the involvement of a server. For a *standalone Blazor WebAssembly app*, assets are deployed as static files to a web server or service capable of serving static content to clients.
 
-IMAGE for Blazor Web Apps
+## Build a native client app with Blazor Hybrid
 
-![XXXXXXXXX](~/blazor/index/_static/blazor-web-app.png)
+*Blazor Hybrid* uses Blazor in a native client app with a blend of native and web technologies. Code runs natively in the .NET process and renders web UI to an embedded Web View control using a local interop channel. WebAssembly isn't used in Hybrid apps. Hybrid apps encompass the following technologies:
```

Comment:
Can we also mention here that Blazor Hybrid enables building an app for the web and mobile & desktop platforms using the same Blazor components?